packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -401,7 +405,54 @@ export type InsertableTreeFieldFromImplicitField<
 	? ApplyKind<InsertableTreeNodeFromImplicitAllowedTypes<Types>, Kind, true>
 	: TSchema extends ImplicitAllowedTypes
 		? InsertableTreeNodeFromImplicitAllowedTypes<TSchema>
-		: unknown;
+		: never;
+
+/**
+ * {@inheritdoc (UnsafeUnknownSchema:type)}
+ * @alpha
+ */
+export const UnsafeUnknownSchema: unique symbol = Symbol("UnsafeUnknownSchema");
+
+/**
+ * A special value which some APIs accept as an alternative to a schema to indicate the schema can not easily be provided at compile time.
```

Comment:
```suggestion
 * A special value which some input APIs accept as an alternative to a schema to indicate the schema can not easily be provided at compile time.
```

Reply:
I don't consider TreeView (well TreeViewAlpha) as an "input Api", yet it accepts this parameter. Perhaps we could say "APIs which are parametric over schema, but if they were parametric over node type instead would not be covariant with respect to that node type", but I'm not sure thats particularly helpful and I think such details might fit better in the remarks section (where I put them).

How about:

```suggestion
 * A special type which can be provided to some APIs as the schema type parameter when schema cannot easily be provided at compile time and an unsafe (instead of disabled) editing API is desired.
```